app/views/fields/active_storage/_preview.html.erb

```diff
@@ -0,0 +1,25 @@
+<div style="width: <%=size[0]%>px; height: auto; overflow: hidden;">
+  <% if attachment.image? %>
+    <% if attachment.variable? %>
+      <%= image_tag(field.variant(attachment, resize_to_limit: size)) %>
```

Comment:
apparently  resize_to_limit does not working with `mini_magic` i think we should change this to `resize` or mention in the docs we are using the  new `image_processing` gem [link](https://github.com/janko/image_processing)

Reply:
@Dreamersoul agreed, however this is unrelated to this PR (`resize_to_limit` was already used before) so how about we merge this PR and address that separately?